.github/workflows/test.yml

```
@@ -41,18 +41,18 @@ jobs:
     steps:
     - name: Install Go
       if: success()
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v3
       with:
-        go-version: 1.17.x
+        go-version: 1.18.x
     - name: Checkout code
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
     - name: Calc coverage
       run: |
         go test ./... -v -covermode=count -coverprofile=coverage.out
     - name: Convert coverage.out to coverage.lcov
-      uses: jandelgado/gcov2lcov-action@v1.0.6
+      uses: jandelgado/gcov2lcov-action@v1.0.9
     - name: Coveralls
-      uses: coverallsapp/github-action@v1.1.2
+      uses: coverallsapp/github-action@v1.1.3
```

```suggestion
      uses: coverallsapp/github-action@1.1.3
```

Seems like coverallsapp changed their versions to not include the `v` now 🤔

---

👍 
Quite a big change from their side 😝